vql/linux/audit/audit_service_test.go

```diff
@@ -0,0 +1,119 @@
+// +build linux
```

Comment:
I had to add a blank line after this for the tests to run

Reply:
Fixed with https://github.com/SUSE/linux-security-sensor/pull/68/commits/1653021fd86fadb648860152634d305ce3f77503